evm/its.js

```diff
@@ -87,6 +87,24 @@ function compare(contractValue, configValue, variableName) {
     }
 }
 
+function isValidDestinationChain(config, chain, destinationChain, originalChain = null) {
```

Comment:
Should I convert chain names to lower case or are we expecting them to be given with the correct case?